test/ConfigProviderTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-aurarouter for the canonical source repository
+ * @copyright Copyright (c) 2015-2018 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive-aurarouter/blob/master/LICENSE.md New BSD License
+ */
+
```

Comment:
no new line before strict_type declaration

Reply:
Have we actually decided on this? The Stratigility repo has no space between the file-level docblock and the declaration, but the zend-expressive repo _does_ (a fact I was going to point out in your review of my latest patch there). We need to decide on a style and stick with one; right now, we have two in as many repos.

Personally, I prefer a line between them, as that makes it clear that the docblock is separate from the declaration.

Reply:
Not sure if we decide it here, but I prefer a new line in between as well.

Reply:
@weierophinney @xtreamwayz I've jest checked, and I was sure that we shouldn't have the empty line there, but I'm really sorry. I was wrong! I prefer the line as well, and here is a previous comment by @weierophinney:
https://github.com/zendframework/zend-expressive-router/pull/41#discussion_r155356898

So lets keep the empty line, update other repositories to add this line, and ignore my all comments about it. Thanks.